zerver/lib/user_groups.py

```diff
@@ -92,6 +92,13 @@ def access_user_group_for_setting(
     return user_group
 
 
+def check_user_group_name(group_name: str) -> str:
+    if len(group_name) > UserGroup.MAX_NAME_LENGTH:
+        raise JsonableError(_("Name too long!"))
```

Comment:
It could be helpful to be more specific about the limit.
Like this example text from the developer documentation: https://github.com/zulip/zulip/blob/179d5cb37d2198a2a50e9903ffd74e436ca1820c/docs/tutorials/new-feature-tutorial.md?plain=1#L464

Reply:
I just took reference from the error message shown for user name. We can probably update that too if this change seems good for user groups.

Reply:
The ideal version of this probably avoids including the number 100 in the string, instead using the variable, to save translation work if the number changes.